drivers/interrupt_controller/intc_plic.c

```diff
@@ -22,12 +22,15 @@
 #include <zephyr/irq.h>
 
 #define PLIC_BASE_ADDR(n) DT_INST_REG_ADDR(n)
+#define PLIC_REG_PRIO_OFFSET 0x0
 #define PLIC_REG_TRIG_TYPE_OFFSET 0x1080
+#define PLIC_REG_IRQ_EN_OFFSET 0x2000
+#define PLIC_REG_REGS_OFFSET 0x200000
```

Comment:
Can you use one single alignment style?

Reply:
These macros will have a single alignment style in the end, is that acceptable? I didnt adopt the existing alignment style as it tends to go out of alignment in the long run